js/main.js

```diff
@@ -208,7 +210,7 @@ var attentionScum = function(vocabulary) {
 var buildQuote = function(vocabulary) {
     var quote = '';
 
-    if (Math.random() < 0.85) {
+    if (_.random(9) < 8) { // 80% chance
```

Comment:
Okay, let's consider some options, because I agree none of these options are ideal. Let's focus on the use case of the 85% randomizer, since that syntax will also handle all the 50% ones. Here are a few options:

```javascript
if (Math.random() < 0.85)
```

* advantages: uses native JS syntax that will likely be readable to any devs who look at it.
* disadvantages: not exactly intuitive, you have to know it returns a float between 0 and 1, and then you can treat that like a percentage chance.

```javascript
if (_.random(1, 100) < 85 )
```

* Advantages: very easy to read and understand
* Disadvantages: Technically more verbose than needed, uses a non-native method that other devs aren't likely to be familiar with (though it seems simple enough to figure out)

```javascript
var d100 = _.random(1, 100);
if (d100 < 85)
```

* Advantages: abstracts the random number generator into a variable, making the if statement easier to understand.
* Disadvantages: again, more verbose than needed, though perhaps that's balanced by being easier to read (like using `if` instead of a ternary).

```javascript
var d100 = Math.random() * 100;
if (d100 < 85)
```

* Advantages: Same as above, but using native JS random number

```javascript
function generatePercentage() { return Math.random() * 100; }
...
var d100 = generatePercentage()
if (d100 < 85)
```

This is the most verbose of all, but has the advantage of moving all the random number crap to a function out of sight, and just setting a local variable in the area that called it. Technically, we don't need the variable, either, but I'm kind of a sucker for this d100 variable.